core/demo/cli_predicate.yaml

```diff
@@ -0,0 +1,15 @@
+pipeline: "BasicBatchPipeline"
+
+inputURI: "csv://core/demo/tiny_predicate.csv"
+
+classifier: "predicate"
+metric: "usage"
+predicate: "=="
```

Comment:
Same, checking for floating point equailty on actual floats is kind of weird, maybe turn this into a slightly more reailstic example? That way we can actually refer to the demos as exemplary documentation of macrobaes best practice.